wps-cloudformation-template.yaml

```diff
@@ -194,6 +231,7 @@ Resources:
         id: 376a0983-1cf3-4145-bafe-96b0750be936
   ComputeEnvironment:
     Type: 'AWS::Batch::ComputeEnvironment'
+    DeletionPolicy: Delete
```

Comment:
Is this not the default already??

Reply:
Nop

Reply:
The environment was not getting deleted automatically. Therefore, I specified it explicitly.

Reply:
! Seems very weird for Cloudformation to leave resources behind... wonder if the reason for this is documented somewhere?